R/tokens.R

```diff
@@ -727,3 +749,18 @@ types.tokens <- function(x) {
     set_types(x) <- value
 }
 
+#' @rdname types
```

Comment:
This needs more documentation to describe concatenator. As part of explaining this as requested below, could you flesh this out more here?

Reply:
We could make this a separate function and describe the behaviour in that Rd entry, rather than in Details in `tokens()`.

Reply:
See [bc0b4ca](https://github.com/quanteda/quanteda/pull/2327/commits/bc0b4ca5510d06868b5b5dc54c942565808f4e52) where I implemented this.